core/db/migrate/20160225152313_remove_tax_rate_from_shipping_rate.rb

```diff
@@ -0,0 +1,29 @@
+class RemoveTaxRateFromShippingRate < ActiveRecord::Migration
+  def up
+    Spree::ShippingRate.find_each do |shipping_rate|
```

Comment:
While I know we don't exactly have a very good track record of it on this project, I would like to be more careful about using models in migrations.

Would you mind adding simple class declarations above?

```ruby
class Spree::ShippingRate < ActiveRecord::Base; end
class Spree::TaxRate < ActiveRecord::Base; end
# etc.
```

That way if for whatever reason we decide to change the class names or remove those classes we don't accidentally break this migration. Alternatively, we could rewrite the migration to use SQL but in this particular case I think that's probably more effort than it's worth.